test/zdtm/static/sock_ip_opts00.c

```diff
@@ -26,7 +26,7 @@ struct sk_opt {
 struct sk_opt sk_opts_v4[] = {
 	{ SOL_IP, IP_FREEBIND, IP_OPT_VAL },
 	{ SOL_IP, IP_PKTINFO, IP_OPT_VAL },
-	{ SOL_IP, IP_TTL, IP_OPT_VAL },
+	{ SOL_IP, IP_TTL, 32 },
```

Comment:
@rahulk789 would you be able to squash this change in the previous commit (`git rebase -i HEAD~2`)?

Reply:
Yup, was a bit busy with work, will do so. Thanks.